### DISCUSSION TITLE
Thank you, Nebula should hire you

### DISCUSSION THREAD

**Discussion by islamahmad** [cpiber/NebulaEnhance#22] [Show and tell]
Thanks a million for this extension, this is for sure a big factor why i will renew my Nebula subscription instead of going back to YouTube 
This extension made my Nebula experience 100000% better, the website is annoying, and the mobile app is even less useful. 
I was forced to download the videos and then watch them on VLC just to have a good enough experience while playing the videos. This extension now fixes a TON of UI issues that I had. 
SHOUT OUT to hide-creator, I'm finally able to annoy that creator that annoyed me to unbelievable levels. Thank you a lot. 
Please give me a way to buy you coffee, or just appreciate you financially

**Comment by islamahmad** [comment]
and another Shout Out from my lungs for the watch queue, thank you thank you, thank you

**Comment by cpiber** [comment]
Thank you for your praise! :)

Nebula actually offered to hire me, but due to personal reasons I had to decline.

I have not setup any way to receive donations, and I currently don't plan to, but I really appreciate the sentiment. It's great that my extension makes the experience more enjoyable.
